services-core/src/economic_viability.rs

```diff
@@ -13,8 +14,9 @@ const GAS_PER_TRADE: f64 = 120_000.0;
 arg_enum! {
     #[derive(Debug)]
     pub enum EconomicViabilityStrategy {
-        Dynamic,
         Static,
+        Dynamic,
+        DynamicBoundedByStatic,
```

Comment:
Could we keep this alphabetical? Or is there a reason for this reordering?

Reply:
I don't have a strong preference either way. Semantically I thought this made sense because it is the same order as we explain the strategies in the readme. There I use this order because the explanation of dynamic depends on static.